content/en/docs/tasks/administer-cluster/running-cloud-controller.md

```diff
@@ -3,17 +3,19 @@ reviewers:
 - luxas
 - thockin
 - wlan0
-title: Kubernetes Cloud Controller Manager
+title: Cloud Controller Manager Administration
 content_template: templates/concept
 ---
 
 {{% capture overview %}}
 
-{{< feature-state state="beta" >}}
+{{< feature-state state="beta" for_k8s_version="v1.11" >}}
 
-Kubernetes v1.6 introduced a new binary called `cloud-controller-manager`. `cloud-controller-manager` is a daemon that embeds cloud-specific control loops. These cloud-specific control loops were originally in the `kube-controller-manager`. Since cloud providers develop and release at a different pace compared to the Kubernetes project, abstracting the provider-specific code to the `cloud-controller-manager` binary allows cloud vendors to evolve independently from the core Kubernetes code.
+{{< glossary_definition term_id="cloud-controller-manager" length="short" >}}
 
-The `cloud-controller-manager` can be linked to any cloud provider that satisfies [cloudprovider.Interface](https://github.com/kubernetes/cloud-provider/blob/master/cloud.go). For backwards compatibility, the [cloud-controller-manager](https://github.com/kubernetes/kubernetes/tree/master/cmd/cloud-controller-manager) provided in the core Kubernetes project uses the same cloud libraries as `kube-controller-manager`. Cloud providers already supported in Kubernetes core are expected to use the in-tree cloud-controller-manager to transition out of Kubernetes core. In future Kubernetes releases, all cloud controller managers will be developed outside of the core Kubernetes project managed by sig leads or cloud vendors.
+Since cloud providers develop and release at a different pace compared to the Kubernetes project, abstracting the provider-specific code to the `cloud-controller-manager` binary allows cloud vendors to evolve independently from the core Kubernetes code.
```

Comment:
nit: Line 16, Is this sentence duplicated from the other related pages?

Reply:
IIRC, it's a direct copy-and-paste.